### PR TITLE
SISRP-13232 - CalCentral Enrollment Card (MVP): back-end and front-end work

### PR DESCRIPTION
**Pull request by christianvuerings**
https://jira.berkeley.edu/browse/SISRP-13232

### CODE REVIEW

File: src/assets/javascripts/angular/controllers/widgets/enrollment/enrollmentCardController.js [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+      },
+      {
+        id: 'adjust',
+        title: 'Adjust'
+      }
+    ]
+  };
+
+  /**
+   * Stop the main stop the main spinner
```
Comment (paulkerschen) [inline src/assets/javascripts/angular/controllers/widgets/enrollment/enrollmentCardController.js]: @christianvuerings Extra words?

Comment (christianvuerings) [inline src/assets/javascripts/angular/controllers/widgets/enrollment/enrollmentCardController.js]: @paulkerschen thank you! Updated the PR.

**Comment by paulkerschen** [comment]
Looks good, @christianvuerings!

**Comment by christianvuerings** [comment]
@paulkerschen thanks for the late Friday evening merge, helps a lot!

**Pull request merged**